### PR TITLE
fix(user managemant): now whenever filtering the table it will be back on page 1

### PR DESCRIPTION
**Pull request by MayaSasson**

### CODE REVIEW

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 1]
```diff
@@ -186,6 +186,10 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
         setFitlerRules(filterRulesToSet)
     }
 
+    useEffect(() => {
+        setPage(1);
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:190]: if the page is already 1, is the fetchUsers called?

Comment (RoeiRom) [reply client/src/components/App/Content/UsersManagement/useUsersManagement.ts:190]: cause when I tried to do it it didnt work

Comment (MayaSasson) [reply client/src/components/App/Content/UsersManagement/useUsersManagement.ts:190]: yes it is

Comment (RoeiRom) [reply client/src/components/App/Content/UsersManagement/useUsersManagement.ts:190]: but isnt the fetch will be called twice? one time due to filter rules change and second time due to page change

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ import axios from 'Utils/axios'
 import { get } from 'Utils/auxiliaryFunctions/auxiliaryFunctions'
 
 import { SortOrderTableHeadersNames } from './UsersManagementTableHeaders'
+import { defaultPage } from './UsersManagement';
```
Comment (roymaniv) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:21]: import order

**Review by ItayBenMoshe — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**